backend/alembic/versions/2023_01_29_1207-7b8f0011e0b0_move_user_streak_from_user_stats_to_.py

```diff
@@ -0,0 +1,31 @@
+"""move user_streak from user_stats to user table
+
+Revision ID: 7b8f0011e0b0
+Revises: 8a5feed819ee
+Create Date: 2023-01-29 12:07:29.379326
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "7b8f0011e0b0"
+down_revision = "49d8445b4c90"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("user", sa.Column("streak_last_day_date", postgresql.TIMESTAMP(), autoincrement=False, nullable=True))
```

Comment:
all timestamps should be `timestamptz`, e.g. `type_=sa.DateTime(timezone=True)` .. this will make sure that we always deal with timezone aware python datetime objects as results from the db (the offset is normally 0 since our DB tz is set to UTC and internally we also use UTC date ..)